charts/open-zaak/README.md

```diff
@@ -64,6 +65,7 @@ table below describes the supported versions
 | `settings.database.password` | The password of PostgreSQL | `"SUPER-SECRET"` |
 | `settings.database.name` | The database name of PostgreSQL | `"open-zaak"` |
 | `settings.database.sslmode` | The SSL-mode used by the postgres client. See [docs](https://www.postgresql.org/docs/current/libpq-ssl.html) for more info | `"prefer"` |
+| `settings.numProxies` | The number of reverse proxies between client and backend container. Set this to 2 if exposing the application through an ingress. This chart deploys one nginx reverse proxy already. | `1`  |
```

Comment:
@bartjkdp this matches the Open Zaak docs, no? If enabling the ingress, you should set it to `2`, otherwise `1` is the correct value.

Reply:
I was refering to https://github.com/open-zaak/charts/pull/24/files#diff-fd1adad8528d3d938e2636b6be48decf821370713cb66b4ca8ed22cdce097777R64

Reply:
This is correct.